external-import/intel471_v2/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  connector-intel471:
+    image: opencti/connector-intel471:6.4.2
```

Comment:
@mmolenda Could you change it and add v2 please to not having problem with v1 of connector ?